modules/monitoring-accessing-alerting-rules-for-all-projects.adoc

```diff
@@ -14,19 +14,21 @@ In a future release, the route to the Thanos Ruler UI will be deprecated in favo
 
 .Prerequisites
 
-* Have the `oc` command installed.
-* Log in as a cluster administrator.
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
 
 .Procedure
 
 . List routes for the `openshift-user-workload-monitoring` namespace:
 +
+[source,terminal]
 ----
 $ oc -n openshift-user-workload-monitoring get routes
 ----
 +
 The output shows the URL for the Thanos Ruler UI:
 +
+[source,terminal]
 ----
 NAME           HOST/PORT
 ...
```

Comment:
There's a spot below this (can't comment on the line in GH directly) where it talks about "Here you can...". Might be better to write the "here" out of it, if possible.

Reply:
I have now completely rewritten this module. We no longer want to point readers to the third-party Thanos Ruler UI. Instead, alerting rules for all projects can be viewed in the Adminstrator perspective in the OCP web console.

As a consequence, the sentence that included the word 'here' has been removed.

I have now resolved this on my local clone and will push the change to my fork shortly.